### ISSUE TITLE
[Bug] Support rename of `user` to `users`

### ISSUE THREAD

**Issue by fivetran-jamie** [fivetran/dbt_lever_source#23]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the issue

In [July 2024](https://fivetran.com/docs/connectors/applications/lever/changelog#july2024), Fivetran renamed the `user` table to `users` for all connectors created after July 27th, 2024. 

Our package only looks for the `user` table but needs to work with this new schema, as it's causing `table not found` errors for folks.

### Relevant error log or model output

```shell
404 Not found: Table fivetran-db:lever.user was not found in location EU; reason: notFound, message: Not found: Table fivetran-data-sandbox-298b:lever.user was not found in location EU
```

### Expected behavior

Should use `users` if it's present, otherwise `user`

### Possible solution

We have a solution for this in a different package right? If I remember correctly @fivetran-joemarkiewicz or @fivetran-catfritz worked on something similar?

### dbt Project configurations

NA quickstart

### Package versions

any

### What database are you using dbt with?

postgres, redshift, snowflake, bigquery, databricks

### How are you running this dbt package?

Fivetran Quickstart Data Model

### dbt Version

na

### Are you willing to open a PR to help address this issue?

- [ ] Yes.
- [ ] Yes, but I will need assistance.
- [ ] No.

**Comment by fivetran-catfritz** [comment]
@fivetran-jamie Yes, very similar to an issue in Recharge. [Here](https://github.com/fivetran/dbt_recharge_source/pull/12) is the PR!

**Comment by fivetran-jamie** [comment]
Perfect! Thank you

**Comment by fivetran-catfritz** [comment]
this is released, so closing out the issue!
